paddle/fluid/operators/detail/grpc_server.cc

```diff
@@ -155,20 +155,22 @@ class RequestPrefetch final : public RequestBase {
 
   void Process() override {
     // prefetch process...
-    std::string varname = request_->OutVarname();
-    VLOG(3) << "RequestPrefetch " << varname;
+    std::string in_var_name = request_->Varname();
+    std::string out_var_name = request_->OutVarname();
+    VLOG(3) << "in_var_name: " << in_var_name
+            << " RequestPrefetch: " << out_var_name;
```

Comment:
Log format should like: `"RequestPrefetch, in_var_name: " << in_var_name << " out_var_name: " << out_var_name`

Reply:
done